contrib/mkimage.sh

```diff
@@ -71,18 +86,18 @@ nameserver 8.8.8.8
 nameserver 8.8.4.4
 EOF
 
-tarFile="$dir/rootfs.tar.xz"
+tarFile="$dir/rootfs.tar${compression}"
```

Comment:
If we change this to just `tarFile="$dir/rootfs.tar${compression:+.$compression}"`, then the only special case we need is something like `[ "$compression" = 'none' ] && compression=''` instead of that set of `if` statements above.